Source/Core/DolphinQt/NANDRepairDialog.cpp

```diff
@@ -48,10 +48,10 @@ NANDRepairDialog::NANDRepairDialog(const WiiUtils::NANDCheckResult& result, QWid
     main_layout->addWidget(warning_label);
 
     std::string title_listings;
-    Core::TitleDatabase title_db;
     const DiscIO::Language language = SConfig::GetInstance().GetCurrentLanguage(true);
     for (const u64 title_id : result.titles_to_remove)
     {
+      Core::TitleDatabase title_db;
```

Comment:
This is a pessimization. The constructor of TitleDatabase reads from disk in its constructor.

Reply:
What does that mean and how do want me to change this?

Reply:
It means you're making the code slower. Please undo the change you made in this file.